node/network/bridge/src/lib.rs

```diff
@@ -127,6 +128,10 @@ pub trait Network: Send + 'static {
 			self.action_sink().send(NetworkAction::WriteNotification(who, peer_set, message)).await
 		}.boxed()
 	}
+
+	/// Ask the network to connect to these nodes and not disconnect from them until removed from the priority group.
+	fn set_priority_group(&self, group_id: String, multiaddresses: HashSet<Multiaddr>) -> Result<(), String>;
+	// TODO: we might want to add `add_to_priority_group` and `remove_from_priority_group` 
```

Comment:
The whole priority groups system makes very little sense to me but we could add functions for tweaking it as needed.